packages/grid/src/features/standard-table/components/StandardTableContent.tsx

```diff
@@ -36,13 +36,29 @@ export const StandardTableContent = React.memo(function StandardTableContent<
 }: Props<TItem, TColumnKey, TColumnGroupKey>) {
   const totalNumColumns = useTotalNumColumns();
 
+  if (errorBanner) {
+    return (
+      <tbody>
+        <tr>
+          <td colSpan={totalNumColumns}>
+            <Spacing num={4} justifyContent={"center"}>
+              <Box alignItems={"center"}>
+                <ResultListBanner bannerState={errorBanner} variant={"error"} />
+              </Box>
+            </Spacing>
+          </td>
+        </tr>
+      </tbody>
+    );
+  }
+
   if (error) {
     return (
       <tbody>
         <tr>
           <td colSpan={totalNumColumns}>
             <Spacing num={4}>
-              <ErrorScreen text={errorLabel || error.message} />
```

Comment:
Maybe we don't need to break the existing API? If `errorLabel` is used by others?